.travis.yml

```diff
@@ -1,21 +1,28 @@
 sudo: false
 language: go
 
-os:
-- linux
-
-env:
-- ARCH=x86_64
-
-go:
-- 1.10.1
-
-script:
-- make test GOFLAGS="-race"
-- make coverage
-- diff -au <(gofmt -d *.go) <(printf "")
-- diff -au <(gofmt -d cmd) <(printf "")
-- diff -au <(gofmt -d pkg) <(printf "")
+matrix:
```

Comment:
If I correctly understood the documentation here https://docs.travis-ci.com/user/multi-os/, it is enough to set os with two architectures:
 
```
os: 
   - linux
   - linux-ppc64le
```

Reply:
I switched to matrix given that the - make test GOFLAGS="-race" is not supported on Power.